apps/ledger-live-desktop/src/renderer/components/DeviceAction/NoSuchAppOnProviderErrorComponent.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { ComponentType } from "react";
+import { ErrorBody } from "../ErrorBody";
+import { Wrapper } from "./rendering";
+import { Trans } from "react-i18next";
+import { Text } from "@ledgerhq/react-ui";
+import ExternalLinkButton from "../ExternalLinkButton";
+import styled from "styled-components";
+import Box from "../Box";
+import { Icons } from "@ledgerhq/react-ui";
+import { urls } from "~/config/urls";
+
+const CtaContainer = styled(Box)`
+  margin-top: 32px;
+`;
+
+const StyledLinkExploreButton = styled(ExternalLinkButton)`
+  display: flex;
+  height: 48px;
+  padding: 8px 24px;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+  border-radius: 500px;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+  color: ${({ theme }) => theme.colors.neutral.c00};
+  background-color: ${({ theme }) => theme.colors.neutral.c100};
+  &:hover {
+    background-color: ${({ theme }) => theme.colors.neutral.c90};
+  }
+`;
+
+const StyledLinkLearnMoreButton = styled(ExternalLinkButton)`
+  margin-top: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+  font-size: 14px;
+  color: ${({ theme }) => theme.colors.neutral.c70};
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+`;
+
+const NoSuchAppOnProviderErrorComponent: React.FC<{
+  error: (Error & { appName: string }) | null | undefined;
+  Icon?: ComponentType<{ color?: string | undefined; size?: number | undefined }>;
+  productName: string;
+}> = ({ error, Icon, productName }) => (
+  <Wrapper id="error-NoSuchAppOnProvider">
+    <ErrorBody
+      Icon={Icon}
```

Comment:
you can do something like that to match with the design :

```
    const theme = useTheme();
    const color = theme.colors.palette.opacityDefault.c05;
    
    top={
      <CircleWrapper color={color} size={72}>
        <Icons.DeleteCircleFill size="L" color="error.c50" />
      </CircleWrapper>
    }
```

Reply:
nice suggestion, is there any theme guideline that i should follow ? there are a lot of ways to do it , starting from pixel-perfect to using the design system (a lot of pallette and themes here..) Thanks for the review, i will change it

Reply:
For the this case color is a assigned like this because CircleWrapper is a styled component. But for other components from the lib you can directly pass color="opacityDefault.c05" for example but they are using the same palette